renku_notebooks/api/amalthea_patches/jupyter_server.py

```diff
@@ -15,7 +15,7 @@ def env(server: "UserServer"):
             "path": "/statefulset/spec/template/spec/containers/0/env/-",
             "value": {
                 "name": "GIT_AUTOSAVE",
-                "value": "1" if server.autosave_allowed else "0",
+                "value": "0",
```

Comment:
This whole patch that adds the GIT_AUTOSAVE environment variable can be removed.